models/pose_estimation_mediapipe/README.md

```diff
@@ -20,6 +22,23 @@ python demo.py
 python demo.py -i /path/to/image -v
 ```
 
+### C++
+
+Install latest OpenCV and CMake >= 3.24.0 to get started with:
+
+```shell
+# A typical and default installation path of OpenCV is /usr/local
+cmake -B build -D OPENCV_INSTALLATION_PATH=/path/to/opencv/installation .
+cmake --build build
+
+# detect on camera input
+./build/opencv_zoo_pose_estimation_mediapipe
+# detect on an image
+./build/opencv_zoo_pose_estimation_mediapipe -m=/path/to/model -i=/path/to/image -v
+# get help messages
+./build/opencv_zoo_pose_estimation_mediapipe -h
+```
+
```

Comment:
Document should be added in `person_detection_mediapipe` folder. The target name is `opencv_zoo_person_detection_mediapipe`

Reply:
oops!